ovs/lib/noderemoval.py

```diff
@@ -93,8 +93,8 @@ def remove_node(node_ip, silent=None):
             storage_router_to_remove_online = True
             for storage_router in storage_router_all:
                 try:
-                    client = SSHClient(storage_router, username='root')
-                except (UnableToConnectException, NotAuthenticatedException):
+                    client = SSHClient(storage_router, username='root', timeout=10)
+                except (UnableToConnectException, NotAuthenticatedException, TimeOutException):
```

Comment:
Proposal: Shuffle responsibility to customer. Let him verify the `offline` nodes and show the reasons why they are detected as offline